### PR TITLE
YapDatabase: Use __autoreleasing keyword where required.

### PR DESCRIPTION
**Pull request by CodeBaseCamp**

### CODE REVIEW

File: YapDatabase/Extensions/FilteredViews/YapDatabaseFilteredViewTransaction.m [hunk 1]
```diff
@@ -875,7 +875,9 @@ - (void)handleInsertObject:(id)object
 	
 	YapDatabaseBlockInvoke blockInvokeBitMask = YapDatabaseBlockInvokeOnInsertOnly;
 	
-	YapDatabaseViewChangesBitMask changesBitMask = YapDatabaseViewChangedObject | YapDatabaseViewChangedMetadata;
+	YapDatabaseViewChangesBitMask changesBitMask =
+		(YapDatabaseViewChangesBitMask)(YapDatabaseViewChangedObject |
+                                    YapDatabaseViewChangedMetadata);
```
Comment (ashabtay) [inline YapDatabase/Extensions/FilteredViews/YapDatabaseFilteredViewTransaction.m]: strange indentation.
Same throughout.

File: YapDatabase/Extensions/Views/YapDatabaseViewTransaction.m [hunk 2]
```diff
@@ -696,7 +696,9 @@ - (BOOL)populateView
 		};
 	}
 	
-	YapDatabaseViewChangesBitMask flags = (YapDatabaseViewChangedObject | YapDatabaseViewChangedMetadata);
+	YapDatabaseViewChangesBitMask flags =
```
Comment (ashabtay) [inline YapDatabase/Extensions/Views/YapDatabaseViewTransaction.m]: indetation

File: YapDatabase/Extensions/FilteredViews/YapDatabaseFilteredViewTransaction.m (hunk 1 above)
Comment (CodeBaseCamp) [inline YapDatabase/Extensions/FilteredViews/YapDatabaseFilteredViewTransaction.m]: Fixed.

Comment (CodeBaseCamp) [inline YapDatabase/Extensions/FilteredViews/YapDatabaseFilteredViewTransaction.m]: Fixed.

File: YapDatabase/Extensions/Views/YapDatabaseViewTransaction.m (hunk 2 above)
Comment (CodeBaseCamp) [inline YapDatabase/Extensions/Views/YapDatabaseViewTransaction.m]: Fixed.

**Comment by CodeBaseCamp** [comment]
Fixed and squashed. Note that the indentation might look weird in github, but it's correct in Xcode, under under assumption that a tab is two blanks.

**Comment by CodeBaseCamp** [comment]
Tried using as little line breaks as possible. Note that some indentations might still look weird in github since YapDatabase uses tabs which under our conventions are displayed as two blanks, but as eight in Github.

**Pull request merged**